static/templates/settings/organization_settings_admin.hbs

```diff
@@ -36,17 +36,17 @@
                 </div>
             </div>
 
+            <label class="dropdown-title">{{admin_settings_label.realm_notifications_stream}}</label>
```

Comment:
Should these `label` elements have a `for` attribute tying them to the `dropdown_list_widget`?  Also, would it be better to have the `dropdown_list_widget` template generate the label elements if the `label` parameter is passed, and not otherwise?  That certainly feels like a cleaner component design.  

@sahil839 @aryanshridhar FYI.

Reply:
Yes would be better to have label inside `dropdown_list_widget`. We currently have label in the `dropdown_list_widget` template only. Just that it has `inline-block` class. So, either we move all such elements to have label on a different line or use some conditional variable to have some labels on same line and some on different.

We can add `for` attribute if we want to open the dropdown on clicking the label otherwise it is fine I think.